website/docs/guides/react.mdx

```diff
@@ -391,7 +391,7 @@ generates:
 >
 > `schema` needs to be your target GraphQL API URL (`"/graphql"` included).
 > `documents` is a glob expression to your `.graphql` files.
-:::
+> :::
```

Comment:
can be remove 
<img width="778" alt="image" src="https://user-images.githubusercontent.com/7361780/162740184-1977e61f-5cf6-4ba2-bc47-45185f5888af.png">